recipes/pasta/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
 
   run:
     - python
-    - dendropy <=4.5.1  # later version do not expose _convert_node_to_root_polytomy
```

Comment:
and this problem is now gone? The missing `_convert_node_to_root_polytomy`?

Reply:
According to https://github.com/smirarab/sepp/pull/126, which has been merged in the current release, the issue should have been solved.